hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/PinotBuildAndPushJob.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.gradle.hadoopdsl.job
+
+
+import com.linkedin.gradle.hadoopdsl.HadoopDslMethod
+import com.linkedin.gradle.hadoopdsl.NamedScope
+
+
+/**
+ * Job class for type=PinotBuildandPush jobs.
+ * <p>
+ * In the example below, the values are NOT necessarily default values; they are simply meant to
+ * illustrate the DSL. Please check that these values are appropriate for your application. In the
+ * DSL, a PinotBuildandPush can be specified with:
+ * <pre>
+ *   pinotBuildPushJob('jobName') {
+ usesTableName 'internalTesting'
+ usesInputPath '/user/input'
+ usesPushLocation 'host:port'
+ *   }
+ * </pre>
+ */
+class PinotBuildAndPushJob extends HadoopJavaJob {
```

Comment:
@jenniferdai Is the code for the PinotBuildAndPushJob actually implemented as an Azkaban Hadoop Java Job? It probably is, I am just double-checking.